common/src/main/java/io/netty/util/internal/logging/Slf4JLoggerFactory.java

```diff
@@ -35,15 +35,11 @@
      */
     @Deprecated
     public Slf4JLoggerFactory() {
-    }
```

Comment:
This is essentially changing behavior of the publicly exposed `INSTANCE` field. Is the motivation for this change just cleanliness or you observed some issues? If it is the former I think it is better to not make this change.

Reply:
@NiteshKant thanks for the comment, I added another INSTANCE to avoid changing the behavior of the publicly exposed 'INSTANCE' field. The main idea about this pr is that I think the instance of 'Slf4JLoggerFactory' in 'InternalLoggerFactory' should a singleton.